EESSI-remove-software.sh

```diff
@@ -0,0 +1,127 @@
+#!/bin/bash
+#
+# Script to remove part of the EESSI software stack (version set through init/eessi_defaults)
+
+# see example parsing of command line arguments at
+#   https://wiki.bash-hackers.org/scripting/posparams#using_a_while_loop
+#   https://stackoverflow.com/questions/192249/how-do-i-parse-command-line-arguments-in-bash
+
+display_help() {
+  echo "usage: $0 [OPTIONS]"
+  echo "  -g | --generic         -  instructs script to build for generic architecture target"
+  echo "  -h | --help            -  display this usage information"
+}
+
+POSITIONAL_ARGS=()
+
+while [[ $# -gt 0 ]]; do
+  case $1 in
+    -g|--generic)
+      echo_yellow ">> GENERIC build requested, taking appropriate measures!"
+      EASYBUILD_OPTARCH="GENERIC"
```

Comment:
This will have zero effect unless it's exported
```suggestion
      export EASYBUILD_OPTARCH="GENERIC"

Reply:
Not sure if it's really needed anyway (it was used in an if statement later on, but we removed that), but I've added `export`.

Reply:
Looking at it again, there's no point in setting it at all, since we're not building anything anyway. So I've removed that line.